packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md

```diff
@@ -0,0 +1,35 @@
+## Guidelines
+
+### General
+
+-   For binary settings, use a [slide toggle](#/styles/form-controls/slide-toggle), [toggle](#/components/FlatSelect), or [checkbox](#/components/Checkbox) instead.
+
+### Labels
+
+-   In most cases, Sliders should have a label.
+
+-   Use the shortest value labels possible, ideally one word (e.g., Low, Medium, High).
+
+### Behavior
+
+-   Adjust the size and behavior of the rail so that users have access to the precision level they need, no more, no less.
+
+-   Always set the lowest value on the left side and the highest on the right.
+
+-   Jumping to evenly spaced marks along the rail may be enabled to help users select discrete values.
+
+-   Discrete values should be selectable by clicking the corresponding value label.
```

Comment:
That reads a bit like the previous line... is there something I'm not seeing? Could the two be combined?

Reply:
Yes. Combining them in the same bullet point would do it, IMO.